src/groovy/au/org/emii/gogoduck/job/JobPresenter.groovy

```diff
@@ -0,0 +1,22 @@
+package au.org.emii.gogoduck.job
+
+/**
+ * Adds in a few extra useful attributes for a client.
+ */
+class JobPresenter extends HashMap {
+    JobPresenter(job, jobExecutorService, jobStoreService, createLink) {
+        super(job.properties)
+
+        if (job.status == Status.NEW) {
```

Comment:
That wouldn't work below though, e.g. if succeed do something, also if succeeded or failed, do something